functest/utils_test.go

```diff
@@ -101,30 +99,39 @@ type rpcStatusResponse struct {
 func createMember(t *testing.T, name string) *user {
 	member, err := newUserWithKeys()
 	require.NoError(t, err)
-	result, err := signedRequest(&root, "CreateMember", name, member.pubKey)
+	member.ref = root.ref
+
+	addBurnAddresses(t)
+
+	result, err := signedRequest(member, "contract.createMember", map[string]interface{}{})
 	require.NoError(t, err)
 	ref, ok := result.(string)
 	require.True(t, ok)
 	member.ref = ref
 	return member
 }
 
-func getBalanceNoErr(t *testing.T, caller *user, reference string) int {
+func addBurnAddresses(t *testing.T) {
+	_, err := signedRequest(&migrationAdmin, "wallet.addBurnAddresses", map[string]interface{}{"burnAddresses": []string{"fake_ba"}})
+	require.NoError(t, err)
+}
+
+func getBalanceNoErr(t *testing.T, caller *user, reference string) *big.Int {
 	balance, err := getBalance(caller, reference)
 	require.NoError(t, err)
 	return balance
 }
 
-func getBalance(caller *user, reference string) (int, error) {
-	res, err := signedRequest(caller, "GetBalance", reference)
+func getBalance(caller *user, reference string) (*big.Int, error) {
+	res, err := signedRequest(caller, "wallet.getBalance", map[string]interface{}{"reference": caller.ref})
```

Comment:
Use `reference` instead of `caller.ref`: sometimes caller and member for balance are different